namespaces/live-1.cloud-platform.service.justice.gov.uk/oasys-keycloak-development/05-certificate.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: oasys-keycloak-development.service.justice.gov.uk
```

Comment:
If you manage with one route-53 zone, you can manage your ingress and cert with some thing like
`development.oasys-keycloak.service.justice.gov.uk`